CreeDictionary/paradigm.py

```diff
@@ -89,11 +90,31 @@ class Heading(StaticCell):
     text = attrib(type=str)
 
 
-Cell = Union[str, StaticCell]
+# frozen=False is a reminder that the inflection is default as None and generated later
+# also inflection related info like inflection frequency in corpus
+@attrs(frozen=False, auto_attribs=True, eq=False)
+class InflectionCell:
+    # the analysis of the inflection (with the lemma to be filled out)
+    # It looks like for example "${lemma}+TAG+TAG+TAG", "TAG+${lemma}+TAG+TAG"
+    analysis: Template
+
+    # the inflection to be generated
+    inflection: Optional[str] = None
+
+    # the frequency of the inflection in the corpus
+    frequency: Optional[int] = None
+
+    def __eq__(self, other):
+        return (
+            isinstance(other, InflectionCell)
+            and self.analysis.pattern == other.analysis.pattern
+            and self.inflection == other.inflection
+        )
```

Comment:
I thought attrs native `eq` (compares attributes with `==` one by one) would do it but no. Who would have thonk string.Template compares by object id rather than its pattern. :thinking: 

this helps the test code here:

```
    assert prefilled_layouts[WordClass.NA, ParadigmSize.BASIC] == [
        [Label("One"), InflectionCell(Template("${lemma}+N+A+Sg"))],
        [Label("Many"), InflectionCell(Template("${lemma}+N+A+Pl"))],
        [Label("Further"), InflectionCell(Template("${lemma}+N+A+Obv"))],
        EmptyRow,
        # TODO: I think there's a mistake with the source material here:
        # This should be a title row, not a label ¯\_(ツ)_/¯
        ["", Heading("Ownership")],
        ["", Heading("One")],
        [Label("my"), InflectionCell(Template("${lemma}+N+A+Px1Sg+Sg"))],
        [Label("your (one)"), InflectionCell(Template("${lemma}+N+A+Px2Sg+Sg")),],
        ["", Heading("Further")],
        [Label("his/her"), InflectionCell(Template("${lemma}+N+A+Px3Sg+Obv"))],
        EmptyRow,
        # TODO: This should be a title, not a heading
        ["", Heading("Smaller/Lesser/Younger")],
        [Label("One"), InflectionCell(Template("${lemma}+N+A+Der/Dim+N+A+Sg"))],
    ]
```

It's used in test code only so I think it's better to be refactored to a test utility function. But that will make this test case a lot uglier.